README.md

```diff
@@ -32,11 +32,17 @@ In the browser navigate to:
 
 ### Importing data
 
+* To import all organisations:
+
 ```bash
-$ rake import:organisation[{department-slug}]
+$ rake import:all_organisations
 ```
 
-Every time you add a new field to the database, you need to delete the existing organisations, and then re-import them.
+* To create or update the content items for an organisation:
+
+```bash
+$ rake import:content_items_by_organisation[{department-slug}]
```

Comment:
Addressed in d01880d: I have added the word `existing` to the description of the rake task, because this is good enough for me.
You said in your first comment, that there is a lot of documentation missing so please, feel free to add it in a separate PR.